documentation/book/assembly-kafka-connect-tls.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-connect-tls-{context}']
+
+= Connecting to Kafka brokers using TLS
+
+Kafka Connect will by default try to connect to Kafka brokers using a plaintext connection.
```

Comment:
Kafka Connect will by default try to connect to Kafka brokers using a plaintext connection.
But it can be configured to use TLS as well.

TO:

By default, Kafka Connect will try to connect to Kafka brokers using a plain text connection. If you would prefer to use TLS additional configuration will be necessary.